src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -684,9 +686,14 @@ public Response ondertekenInformatieObject(
     ) {
         final EnkelvoudigInformatieObject enkelvoudigInformatieobject = drcClientService.readEnkelvoudigInformatieobject(
```

Comment:
Some coverage by a unit and/or integration test would be great of course. Not sure if that is feasible in the time left for this sprint..?

Reply:
Fair point, I'll give it a go

Reply:
I added an integration test :)